doc/modules/con_Approval.adoc

```diff
@@ -3,7 +3,7 @@
 [id="Approval"]
 = Working with approval
 
-Catalog supports multi-level approval workflows, providing the flexibility to create a tiered administrative process that includes selected organizational groups. Administrators can design approval workflows, view request details, approve or deny requests, and monitor status of existing requests. Once created, an approval workflow can be assigned to Catalog portfolios.
+Catalog supports group-based approval workflows, providing the flexibility that includes selected organizational groups. Administrators can assign approval groups, view request details, approve or deny requests, and monitor status of existing requests. Once created, an approval workflow can be assigned to Catalog portfolios.
```

Comment:
Approval workflows can currently be applied to Portfolios, Products and Platform Inventories.  The intention is to support Platforms as well, but is dependent on tagging support on the Sources platform service which is in-progress.

Reply:
Thanks @gmcculloug, I've added this detail to the doc. For the time I've left platforms in there since its in progress.